From 562b1b91942d1157efcd98c7e2e01be98a2f0e3b Mon Sep 17 00:00:00 2001
From: zhaochen <zhaochen@codeaurora.org>
Date: Fri, 21 Aug 2020 17:41:33 +0800
Subject: [PATCH] soc: qcom: watchdog: Modify some debug level

Modify the irq_count print debug level from info to debug.

Change-Id: I1c5677414b2a97fc2697b003620a80d8843589a7
Signed-off-by: zhaochen <zhaochen@codeaurora.org>
Signed-off-by: Sevenrock <sevenrock@hotmail.de>
---
 drivers/soc/qcom/qcom_wdt_core.c | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/drivers/soc/qcom/qcom_wdt_core.c b/drivers/soc/qcom/qcom_wdt_core.c
index 5d9b152f0249..1e774e4fdca6 100644
--- a/drivers/soc/qcom/qcom_wdt_core.c
+++ b/drivers/soc/qcom/qcom_wdt_core.c
@@ -119,24 +119,24 @@ static void print_irq_stat(struct msm_watchdog_data *wdog_dd)
 	int cpu;
 	struct qcom_irq_info *info;
 
-	pr_info("(virq:irq_count)- ");
+	pr_debug("(virq:irq_count)-\n");
 	for (index = 0; index < NR_TOP_HITTERS; index++) {
 		info = &wdog_dd->irq_counts[index];
-		pr_cont("%u:%u ", info->irq, info->total_count);
+		pr_debug("%u:%u\n", info->irq, info->total_count);
 	}
-	pr_cont("\n");
+	pr_debug("\n");
 
-	pr_info("(cpu:irq_count)- ");
+	pr_debug("(cpu:irq_count)-\n");
 	for_each_possible_cpu(cpu)
-		pr_cont("%u:%u ", cpu, wdog_dd->tot_irq_count[cpu]);
-	pr_cont("\n");
+		pr_debug("%u:%u\n", cpu, wdog_dd->tot_irq_count[cpu]);
+	pr_debug("\n");
 
-	pr_info("(ipi:irq_count)- ");
+	pr_debug("(ipi:irq_count)-\n");
 	for (index = 0; index < NR_IPI; index++) {
 		info = &wdog_dd->ipi_counts[index];
-		pr_cont("%u:%u ", info->irq, info->total_count);
+		pr_debug("%u:%u\n", info->irq, info->total_count);
 	}
-	pr_cont("\n");
+	pr_debug("\n");
 }
 
 static void compute_irq_stat(struct work_struct *work)
-- 
2.39.2

